scripts/exregional_make_grid.sh

```diff
@@ -266,29 +266,36 @@ generation executable (exec_fp):
 # namelist file.
 #
   settings="
-'regional_grid_nml': {
-    'plon': ${LON_CTR},
-    'plat': ${LAT_CTR},
-    'delx': ${DEL_ANGLE_X_SG},
-    'dely': ${DEL_ANGLE_Y_SG},
-    'lx': ${NEG_NX_OF_DOM_WITH_WIDE_HALO},
-    'ly': ${NEG_NY_OF_DOM_WITH_WIDE_HALO},
-    'pazi': ${PAZI},
- }
+'regional_grid_nml':
+    'plon': ${LON_CTR}
+    'plat': ${LAT_CTR}
+    'delx': ${DEL_ANGLE_X_SG}
+    'dely': ${DEL_ANGLE_Y_SG}
+    'lx': ${NEG_NX_OF_DOM_WITH_WIDE_HALO}
+    'ly': ${NEG_NY_OF_DOM_WITH_WIDE_HALO}
+    'pazi': ${PAZI}
 "
-#
-# Call the python script to create the namelist file.
-#
-  ${USHdir}/set_namelist.py -q -u "$settings" -o ${rgnl_grid_nml_fp} || \
-    print_err_msg_exit "\
-Call to python script set_namelist.py to set the variables in the
-regional_esg_grid namelist file failed.  Parameters passed to this script
-are:
-  Full path to output namelist file:
-    rgnl_grid_nml_fp = \"${rgnl_grid_nml_fp}\"
-  Namelist settings specified on command line (these have highest precedence):
-    settings =
-$settings"
+
+  # Store the settings in a temporary file
```

Comment:
Any chance we can get this utility to accept a dictionary so we don't have to use hacky temporary files? Or will I have to open an issue for that? 😉

Reply:
`uw` supports read-from-stdin / write-to-stdout for many operations, e.g.
```
$ export VAL1=foo
$ export VAL2=88
$ (cat <<EOF
namelist:
  key1: $VAL1
  key2: $VAL2
EOF
 ) | uw config realize --input-format yaml --output-format nml
```
emits
```
&namelist
    key1 = 'foo'
    key2 = 88
/
```
to stdout. Or use `-o` / `--output-file` to write directly to a file (`--output-format nml` is unnecessary if the output filename extension is `.nml`).

Reply:
Paul, I appreciate your input here. I am happy to make that change. @mkavulich would you prefer the temporary file or the stdin version?

Reply:
I do prefer the stdin version just because the tempfile seems pretty hacky, especially as multiple calls result in multiple temporary files that may not be properly cleaned up in the case of an error.